source/docs/software/advanced-gradlerio/profiling-with-visualvm.rst

```diff
@@ -0,0 +1,176 @@
+Profiling with VisualVM
```

Comment:
Somewhere this should explicitly say Java. It's implied by saying JVM, but I think should be more explicit.